ui/mirage/config.js

```diff
@@ -312,6 +313,65 @@ export default function() {
   this.get('/client/allocation/:id/stats', clientAllocationStatsHandler);
   this.get('/client/fs/logs/:allocation_id', clientAllocationLog);
 
+  this.get('/client/fs/ls/:allocation_id', (schema, { queryParams }) => {
+    if (queryParams.path.endsWith('empty-directory')) {
+      return [];
+    } else if (queryParams.path.endsWith('directory')) {
+      return [
+        {
+          Name: 'another',
+          IsDir: true,
+          ModTime: moment().format(),
+        },
+      ];
+    } else if (queryParams.path.endsWith('another')) {
+      return [
+        {
+          Name: 'something.txt',
+          IsDir: false,
+          ModTime: moment().format(),
+        },
+      ];
+    } else {
+      return [
+        {
+          Name: '🤩.txt',
+          IsDir: false,
+          Size: 1919,
+          ModTime: moment()
+            .subtract(2, 'day')
+            .format(),
+        },
+        {
+          Name: '🙌🏿.txt',
+          IsDir: false,
+          ModTime: moment()
+            .subtract(2, 'minute')
+            .format(),
+        },
+        {
+          Name: 'directory',
+          IsDir: true,
+          Size: 3682561,
+          ModTime: moment()
+            .subtract(1, 'year')
+            .format(),
+        },
+        {
+          Name: 'empty-directory',
+          IsDir: true,
+          ModTime: moment().format(),
+        },
+      ];
+    }
+  });
+
+  this.get('/client/fs/stat/:allocation_id', (schema, { queryParams }) => {
+    return {
+      IsDir: !queryParams.path.endsWith('.txt'),
+    };
+  });
```

Comment:
Below these `/client` handlers is a `HOSTS.forEach` block that serves to also include these client handles on client IPs, since in some places (not these two places) we try to connect to the client directly first.

Even though we don't plan on connecting to clients directly for these end points since they are lightweight, I think it's worth putting these handlers in both places just for completeness sake.

Reply:
👍🏻 makes sense, done, thanks.